sdk/keyvault/azure-security-keyvault-certificates/src/private/certificate_key_usage.hpp

```diff
@@ -1,40 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-/**
- * @file
- * @brief Key Usage constants.
- *
- */
-#pragma once
-#include <azure/keyvault/certificates/certificate_client_models.hpp>
-
-namespace Azure { namespace Security { namespace KeyVault { namespace Certificates {
-
-  const CertificateKeyUsage CertificateKeyUsage::DigitalSignature(_detail::DigitalSignatureValue);
```

Comment:
Do customers need these to be public?

Reply:
All of them are publicly declared, in a header.
This file does only instantiate these statics which should not normally be done in an .hpp file.
But this file ends up being included only in exactly one .cpp file, so it turns out ok.
You can see this file is a private header in /src, so customers could not have get to it anyway.
And I moved all these statics instantiations to a corresponding .cpp file - newly created for this certificate_client_models.cpp